src/gen_modules_partitioning.F90

```diff
@@ -152,10 +160,8 @@ subroutine par_ex(COMM, mype, abort)       ! finalizes MPI
   
   if (mype==0) print *, 'FESOM calls MPI_Finalize'
   call MPI_Finalize(error)
-#endif
-         ! oifs/echam
-#endif
-         ! oasis
+#endif ! oifs/echam
```

Comment:
I don't think all compiler will allow comments on the end of preprocessor statements. This would be reverted